### PR TITLE
fix json parse failure when only 1 MAC address is returned.

### PR DESCRIPTION
**Pull request by tao-sj**
add API func for CDP neighbors.
add show mac address table and show cdp neighbors to client test code.

### CODE REVIEW

File: pkg/client/sysinfo_test.go [hunk 1]
```diff
@@ -53,15 +53,15 @@ func TestParseShowVersionJsonOutput(t *testing.T) {
...
-                {
-                        input: "show.version.4",
-                        exp: &SysInfo{
-                                ChassisID: "Nexus9000 C93180YC-FX Chassis",
-                                CPUName:   "Intel(R) Xeon(R) CPU D-1528 @ 1.90GHz",
-                        },
-                        shouldFail: false,
-                        shouldErr:  false,
-                },
+		{
```
Comment (tao-sj) [inline pkg/client/sysinfo_test.go:56]: change in this file is from gofmt.

**Comment by greenpau** [comment]
@tao-sj , is this ready to go?

**Comment by tao-sj** [comment]
> @tao-sj , is this ready to go?

yes!

**Comment by greenpau** [comment]
@tao-sj , 👍 released https://github.com/greenpau/go-cisco-nx-api/releases/tag/v1.0.4

**Comment by tao-sj** [comment]
> @tao-sj , 👍 released https://github.com/greenpau/go-cisco-nx-api/releases/tag/v1.0.4

Thanks for your prompt action!

**Pull request merged**